samila/functions.py

```diff
@@ -110,3 +110,29 @@ def nft_storage_upload(api_key, data):
         result["status"] = False
         result["message"] = str(e)
         return result
+
+
+def samila_help():
+    """
+    Print samila details.
+
+    :return: None
+    """
+    print(OVERVIEW)
+    print("Repo : https://github.com/sepandhaghighi/samila")
+
+
+def isSameData(data1, data2, precision=10**-5):
```

Comment:
Please use the same format as other functions, something like `is_same_data` or `is_equal` 💯

Reply:
Ah You're right.